drivers/tty/serdev/pibridge.c

```diff
@@ -382,6 +382,8 @@ int pibridge_req_io(u8 addr, u8 cmd, u8 *snd_buf, u16 snd_len, u8 *rcv_buf,
 		    u16 rcv_len)
 {
 	struct pibridge_pkthdr_io pkthdr;
+	u16 to_receive;
+	u16 to_discard;
```

Comment:
These can be declared `u8`, right? IO packets can have a maximum length of 31 bytes.

Actually, why did you use `u16` in `pibridge_req_gate_tmt()`? Gate packets have a max length of 255 bytes. I guess you chose that for consistency with the length passed into the function. Maybe the function signature should be changed to use `u8` for length everywhere?

Reply:
I agree, the signature should be changed then. Also for io packet case.